### DISCUSSION TITLE
动画时机是否可以运行时修改

### DISCUSSION THREAD

**Discussion by tiny-lin** [Tencent/libpag#1950] [Q&A] [answered]
使用场景：Web

遇到的问题：
目前pag文档中只有文本和图片的替换，请教一下，视频里歌词滚动的时机可以在运行时替换么？主要的目的是想用一份pag模版播放不同的歌曲，不同歌曲的歌词时间是不同的

https://github.com/Tencent/libpag/assets/43535625/a93664a3-19d2-4840-9667-7f8138fc92bf

**Comment by kevingpqi123** [comment, marked as answer]
PAGLayer 上面有一个 setStartTime 的接口，可以任意设置图层的开始显示时间（设置的时间相对于的是父级图层时间轴），看下是否可以满足你的需求
